lib/order.rb

```diff
@@ -0,0 +1,52 @@
+require_relative 'menu'
+
+class Order
+  
+  attr_reader :menu, :basket
+
+  def initialize
+  	@menu = Menu.new
+  	@basket = []
+  	@total_amount = 0
+  	@dish = nil
+  	@quantity = 0
+  end
+
+  def add(dish, quantity = 1)
+  	@dish = dish.downcase
```

Comment:
This places a dependency on the dish to respond to `.downcase`. What if you make a dish class? Or even just use symbols?